androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt

```diff
@@ -248,4 +309,116 @@ class ComposePrototypeUITest : BaseTests() {
         }
         bye()
     }
+
+    @Test
+    fun testUnknownAgent() {
+        apiHelper.disconnectAllConversations()
+        enterDeploymentInfo(testConfig.humanizeDisableDeploymentId)
+        connect()
+        sendMsg(helloText)
+        val conversationInfo = apiHelper.answerNewConversation()
+        if (conversationInfo == null) AssertionError("Unable to answer conversation.")
+        else {
+            Log.i(TAG, "Conversation started successfully.")
+            apiHelper.sendOutboundMessageFromAgentToUser(conversationInfo, outboundMessage)
+            verifyResponse(outboundMessage)
+            verifyResponse(humanizeDisabledText)
+            verifyResponse(humanText)
+            apiHelper.sendConnectOrDisconnect(conversationInfo, false, true)
```

Comment:
We can use kotlin default argument feature here. You are already doing it with `wrapup` value
In most cases `sendConnectOrDisconnect` has values of `connecting=false` and `wrapup=true`. 
SOmething like that: 
` fun API.sendConnectOrDisconnect(conversationInfo: Conversation, connecting: Boolean = false, wrapup: Boolean = true)` 
can help us to get rid of all this repetitions. 
If you decide to leave it as it is now, we should add argument names for this booleans - otherwise it is confusing to understand which boolean does what.

Reply:
Yes, I can fix this. Thanks for suggestion.